### ISSUE TITLE
cataclysm-dda olny english language again.

### ISSUE THREAD

**Issue by SallyGordon** [NixOS/nixpkgs#186304]
### Describe the bug
stable version and git(override to current commit) version.

### Steps To Reproduce
install cataclysm-dda.

### Expected behavior
no non-english language.

### Screenshots
<img width="870" alt="image" src="https://user-images.githubusercontent.com/111138303/184387376-a6a8c35e-850f-45a4-ba8e-08dbd91a5174.png">

### Notify maintainers

@mnacamura @DeeUnderscore
### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"aarch64-darwin"`
 - host os: `Darwin 22.0.0, macOS 13.0`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.10.3`
 - channels(root): `"nixpkgs"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixpkgs`
```

**Comment by SallyGordon** [comment]
The old solution https://github.com/NixOS/nixpkgs/pull/74064.
Current not work:
https://github.com/NixOS/nixpkgs/blob/ff8e8d4b4f7c934e16546beefce9e547b6b65d5c/pkgs/games/cataclysm-dda/common.nix#L45-L49

**Comment by DeeUnderscore** [comment]
Hey, I opened a PR that might fix this at #186521. If you are able to build the PR under Darwin and see if that helps, it would be appreciated.

**Comment by SallyGordon** [comment]
@DeeUnderscore Thank you!
